portal/server/generateApp.ts

```diff
@@ -97,7 +116,7 @@ const generateApp = () => {
   app.use(copyCsrfTokenFromQueryToBody());
   app.use(csrf({
     cookie: {
-      ...cookie,
+      ...cookie as csrf.CookieOptions,
```

Comment:
You can remove the need for this explicit cast by setting the type of `cookie` to `session.CookieOptions & csrf.CookieOptions` (I think it's best to avoid explicit casts like this where possible - the most appropriate use case for it that I've found is in test files)